### PR TITLE
docs: fix broken link

### PR DESCRIPTION
**Pull request by pdecat**
Link to Manifest Annotations is broken

### CODE REVIEW

File: docs/CLI/manifest_config.mdx [hunk 1]
```diff
@@ -111,7 +111,7 @@ Additionally, the latest version of `dockerd` recognizes OCI manifests and does
 ## Push / Pull config
 
 As mentioned in the beginning of this doc, the manifest config is not used by `oras` and is not worth pulling in most scenarios.
```
Comment (TerryHowe) [inline docs/CLI/manifest_config.mdx:113]: I'm a little confused because this link seems to be working for me https://oras.land/docs/cli/manifest_config/

Comment (pdecat) [reply docs/CLI/manifest_config.mdx:113]: Not for me https://oras.land/docs/cli/manifest_config/manifest_annotations

Comment (pdecat) [reply docs/CLI/manifest_config.mdx:113]: Talking about this one 
Screenshot_20230624-214948

Comment (TerryHowe) [reply docs/CLI/manifest_config.mdx:113]: Ah, works for https://oras.land/docs/CLI/manifest_config/ but not for https://oras.land/docs/CLI/manifest_config

Comment (pdecat) [reply docs/CLI/manifest_config.mdx:113]: Doesn't work for me in either case 😕

Comment (TerryHowe) [reply docs/CLI/manifest_config.mdx:113]: there is some odd url rewrite going on with https://oras.land/docs/CLI/manifest_config with a lower case "cli"

File: docs/CLI/manifest_config.mdx [hunk 2]
```diff
@@ -111,7 +111,7 @@ Additionally, the latest version of `dockerd` recognizes OCI manifests and does
...
-However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](./manifest_annotations).
+However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](../manifest_annotations).
```
Comment (TerryHowe) [inline docs/CLI/manifest_config.mdx:114]:
```suggestion
However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](/cli/manifest_annotations).
```

should work in either case

**Review by TerryHowe — APPROVED** [review]
/lgtm

I'd like Asmit's opinion here

File: docs/CLI/manifest_config.mdx [hunk 3]
```diff
@@ -111,7 +111,7 @@ Additionally, the latest version of `dockerd` recognizes OCI manifests and does
 ## Push / Pull config
 
 As mentioned in the beginning of this doc, the manifest config is not used by `oras` and is not worth pulling in most scenarios.
-However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](./manifest_annotations).
+However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](/cli/manifest_annotations).
```
Comment (asmitbm) [inline docs/CLI/manifest_config.mdx:114]:
```suggestion
However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](./manifest_annotations).
```
PR #194 should fix all the links after it's merged.

**Review by asmitbm — CHANGES_REQUESTED** [review]

File: docs/CLI/manifest_config.mdx (hunk 1 above)
Comment (asmitbm) [reply docs/CLI/manifest_config.mdx:113]: Yeah, but both the links for me. I have updated the changes in PR #194. Thanks for noticing this @pdecat :D

**Pull request closed without merging**